app/services/reports/adult_incontinence_report_service.rb

```diff
@@ -91,5 +101,54 @@ def donated_supplies
                                     .where(itemizable: organization.donations.for_year(year))
                                     .sum(:quantity)
     end
+
+    def distributed_adult_incontinence_items_from_kits
+      organization_id = @organization.id
+      year = @year
+
+      sql_query = <<-SQL
+        SELECT SUM(line_items.quantity * kit_line_items.quantity)
+        FROM distributions 
+        INNER JOIN line_items ON line_items.itemizable_type = 'Distribution' AND line_items.itemizable_id = distributions.id 
+        INNER JOIN items ON items.id = line_items.item_id 
+        INNER JOIN kits ON kits.id = items.kit_id 
+        INNER JOIN line_items AS kit_line_items ON kits.id = kit_line_items.itemizable_id
+        INNER JOIN items AS kit_items ON kit_items.id = kit_line_items.item_id
+        INNER JOIN base_items ON base_items.partner_key = kit_items.partner_key 
+        WHERE distributions.organization_id = ?
+          AND EXTRACT(year FROM issued_at) = ?
+          AND LOWER(base_items.category) LIKE '%adult%'
+          AND NOT (LOWER(base_items.category) LIKE '%cloth%' OR LOWER(base_items.name) LIKE '%cloth%')
+          AND kit_line_items.itemizable_type = 'Kit';
+      SQL
+
+      sanitized_sql = ActiveRecord::Base.send(:sanitize_sql_array, [sql_query, organization_id, year])
+
+      result = ActiveRecord::Base.connection.execute(sanitized_sql)
+
+      result.first['sum'].to_i
+    end
+
+    def adults_served_per_month
+      total_people_served_with_loose_supplies_per_month + total_people_served_with_supplies_from_kits_per_month
+    end
+
+    def total_people_served_with_loose_supplies_per_month
+      organization
+        .distributions
+        .for_year(year)
+        .joins(line_items: :item)
+        .merge(Item.adult_incontinence)
+        .sum('line_items.quantity / COALESCE(items.distribution_quantity, 50)') / 12
+    end
+
+    def total_people_served_with_supplies_from_kits_per_month
+      organization
+      .distributions.for_year(year)
+      .joins(line_items: {item: :kit})
+      .merge(Item.adult_incontinence)
+      .where.not(items: {kit_id: nil})
+      .sum('line_items.quantity / COALESCE(items.distribution_quantity, 50)') / 12
+    end
```

Comment:
Per the issue,  "
The number of adults assisted with kit items is the number of kits that contain adult incontinence supplies distributed divided by the "quantity per individual" on the kit item. if there is no quantity per individual provided, assume that quantity is 1."

Reply:
Quick question, also I apologize for missing that bit! Is it a safe assumption that the distribution_quantity is equal to the "quantity per individual"?

Reply:
Yes.  Sorry for the terminology ambiguity -- they are the same.